simple_history/models.py

```diff
@@ -179,7 +179,8 @@ def copy_fields(self, model):
                 if isinstance(old_field.rel.to, str) and old_field.rel.to == 'self':
                     object_to = old_field.model
                 else:
-                    object_to = old_field.rel.to
+                    #           required for Django <= 1.8                       # required for Django >= 2.0
+                    object_to = old_field.rel.to if hasattr(old_field, 'rel') else old_field.remote_field.model
```

Comment:
As said before, this is the wrong order to avoid warnings.

Line 179 seems to have the same problem.

Reply:
So, you want to see  
```old_field.remote_field.model if hasattr(old_field, 'remote_field') else old_field.rel.to```
?

Reply:
Did that edit plus helped out @flesser 's merge to guard new uses of `old_field.rel.to` here https://github.com/flesser/django-simple-history/pull/4 .